### DISCUSSION TITLE
How to get started

### DISCUSSION THREAD

**Discussion by neumann1966** [Motion-Project/motionplus#54] [Q&A]
I have used motion for many years now. And love it.

I have started a new server and saw that motion plus is now an option.

But I'm not a linux specialist, and is not familiar with compiling code myself.

When I started with motion, I just installed a package with apt-get at the time, now it's just apt, is that an option with motion plus?

I followed the compilation example, but don't really know what to do next. I think I would be better off with just installing the precompiled package if possible. Is there even a precompiled package?

Otherwise is there somewhere I can read how to get morion plus working. I have my old camera conf files from motion that I was hoping could be reused so it would be an fix to get motion plus running instead off motion.

Or would I be better off by reinstalling motion on the new server, and just keep using motion, being a totally noob in compiling regards?

**Comment by Jachimo** [comment]
I am new to Motionplus (MotionPlus? Motion+?) as well, but I don't *think* that you need to compile it yourself. 

Currently I am just working from the release `.deb` file and it seems to install fine on Raspbian Bullseye.  There were some prerequisites that I had to install, but once those were taken care of, the basic functionality worked well.

**Comment by neumann1966** [comment]
Wow, you are absolutely right. I made a go at it again, and got it up 
and running.

Only thing now is I can't see more that 1 camera out of 4. All cameras 
are ip cameras, and have been working flawlessly on motion.

I have conf files for each of the cameras, they are all looking the 
same, except the url, which is obviously different from camera to 
camera.

Even though only camera 1 is showing picture. The others are on the 
configuration page by numbers (camera 1, camera 2, camera 3 and camera 
4. But only camera 1 is showing stream.

Any ideas?

James skrev den 14-05-2023 16:58:

> I am new to Motionplus (MotionPlus? Motion+?) as well, but I don't 
> _think_ that you need to compile it yourself.
> 
> Currently I am just working from the release .deb file and it seems to 
> install fine on Raspbian Bullseye. There were some prerequisites that I 
> had to install, but once those were taken care of, the basic 
> functionality worked well.
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/Motion-Project/motionplus/discussions/54#discussioncomment-5897441
[2]

**Reply by Jachimo** [reply]
I don't have any particular solutions, but I would try and see whether the issue is with MotionPlus only loading one camera (for whatever reason), or if there's only one of the four cameras that it will show video from.

I.e., if you remove the conf files for 3 of the 4 cameras and only leave 1 in the configuration directory, does it work regardless of which camera's conf file is there?  In that case, then I think you must have something misconfigured.  But if only one of the 3 cameras works even when you try to use each one individually, then I would try to drill into what makes the one working camera different from the 3 non-working cameras.  There's almost certainly something different about the data stream coming from that one camera, if I had to guess.

**Comment by neumann1966** [comment]
I made it working yesterday. Yeah.

It seems you can't just use the camerafiles from motion, but have to conform to the newer format of motionplus. That did the trick. 😀

Den 18. maj 2023 23.10.09 CEST, James ***@***.***> skrev:
>I don't have any particular solutions, but I would try and see whether the issue is with MotionPlus only loading one camera (for whatever reason), or if there's only one of the four cameras that it will show video from.
>
>I.e., if you remove the conf files for 3 of the 4 cameras and only leave 1 in the configuration directory, does it work regardless of which camera's conf file is there?  In that case, then I think you must have something misconfigured.  But if only one of the 3 cameras works even when you try to use each one individually, then I would try to drill into what makes the one working camera different from the 3 non-working cameras.  There's almost certainly something different about the data stream coming from that one camera, if I had to guess.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/Motion-Project/motionplus/discussions/54#discussioncomment-5942369
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
-- 
Dette er sendt fra min mobiltelefon. Undskyld at jeg fatter mig i korthed.

**Reply by featherstonej** [reply]
I am setting up my cameras for the first time. Would you be willing to share your config files? My cameras show up, but they say Unable to connect to the camera. I can connect to them using VLC on my phone with the same url, so I am sure it has something to do with the settings I am using or that I am not using. Thanks.
